Tests/MongoSwiftSyncTests/SyncChangeStreamTests.swift

```diff
@@ -692,14 +737,22 @@ final class SyncChangeStreamTests: MongoSwiftTestCase {
     // TODO: SWIFT-576: Implement prose tests 14, 17, & 18
 
     func testChangeStreamOnAClient() throws {
-        guard MongoSwiftTestCase.topologyType != .single else {
-            print(unsupportedTopologyMessage(testName: self.name))
-            return
-        }
-
         let client = try MongoClient.makeTestClient()
-        guard try client.serverVersion() >= ServerVersion(major: 4, minor: 0) else {
-            print("Skipping test case for server version \(try client.serverVersion())")
+        let testRequirements = TestRequirement(
+            minServerVersion: ServerVersion(major: 4, minor: 0),
+            acceptableTopologies: [.replicaSet, .sharded]
+        )
+
+        let unmetRequirements = try client.checkRequirements(testRequirements)
+        guard unmetRequirements == nil else {
+            switch unmetRequirements {
+            case .minServerVersion:
+                print("Skipping test case for server version \(try client.serverVersion())")
```

Comment:
this one looks like it can just use the regular print method too.

Reply:
ok